extensions/amp-accordion/1.0/accordion.js

```diff
@@ -97,16 +104,82 @@ export function Accordion({
     [expandSingleSection]
   );
 
+  const isExpanded = useCallback(
+    (id, defaultExpanded) => expandedMap[id] ?? defaultExpanded,
+    [expandedMap]
+  );
+
+  const toggle = useCallback(
+    (id) => {
+      if (id) {
+        if (hasOwn(expandedMap, id)) {
```

Comment:
They can only be possibly "own" properties here. So either `id in expandedMap` or `expandedMap[id] != null` are totally functional in this case.

Reply:
Thank you, very helpful, updated in the latest!